src/mapping/epoch-action.ts

```diff
@@ -40,3 +40,35 @@ export function handleEpochUnstakeSuccess(
     entity.save();
   }
 }
+
+export function handleEpochCleanup(
+  data: TypedMap<string, JSONValue>,
+  receipt: near.ReceiptWithOutcome
+): void {
+  const timestamp = receipt.block.header.timestampNanosec;
+
+  const stakeAmountToSettle = BigInt.fromString(data.get('stake_amount_to_settle')!.toString());
+  const unstakeAmountToSettle = BigInt.fromString(data.get('unstake_amount_to_settle')!.toString());
+
+  // Use epochId as id
+  const id = receipt.block.header.epochId.toBase58();
+
+  const entity = EpochCleanup.load(id);
+  if (entity) {
+    if (entity.stakeAmountToSettle.equals(stakeAmountToSettle) &&
+        entity.unstakeAmountToSettle.equals(unstakeAmountToSettle)
+    ) {
+      // do nothing
```

Comment:
Since the code block is empty, we can move the `else` condition (`!entity.stakeAmountToSettle.equals(stakeAmountToSettle) || !entity.unstakeAmountToSettle.equals(unstakeAmountToSettle)`) into `if` condition.